solidus_globalize.gemspec

```diff
@@ -25,7 +25,6 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'friendly_id-globalize'
   s.add_runtime_dependency 'globalize', '~> 5.0.1'
-  s.add_runtime_dependency 'spree_i18n', '~> 3.1.0.beta'
```

Comment:
mmm...here, discussing with @alepore we're thinking if it's better use solidus_i18n from github master considering the edge status of both gems. What is your idea?